src/App.js

```diff
@@ -11,8 +15,12 @@ class App extends React.PureComponent {
   render() {
     return (
       <>
-        <h1>Calculator Page</h1>
-        <Calculator />
+        <Navbar />
+        <Routes>
+          <Route path="/home" element={<Home />} />
+          <Route path="calculator" element={<Calculator />} />
+          <Route path="about" element={<Quote />} />
```

Comment:
- [ ] [OPTIONAL] Great job 👏 . It will be best if your route `path`s are:

```
path="/"
path="/calculator"
path="/about"
```

This is so that the homepage displays once the website is loaded. Right now the `home` page displays only when it is clicked. Kindly fix this 🙏

Reply:
@Ugboaja-Uchechi  Sure, thanks for the feedback 😄